src/averageAges.js

```diff
@@ -14,12 +14,22 @@
  *
  * @return {number}
  */
+
 function calculateMenAverageAge(people, century) {
   // write code here
   // learn how to use array methods like .filter .map .some .every .find .reduce
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  let men = people.filter(person => person.sex === 'm');
+
+  men = century ? men.filter(person =>
+    Math.ceil(person.died / 100) === century) : men;
+
+  const ages = men.map(p => p.died - p.born);
+
+  return Math.round(ages.reduce((a, b) => a + b) / ages.length * 100) / 100;
```

Comment:
You can move it to separate function to avoid code dublication

Reply:
for 2 lines of code - not a big deal, but okay, done